src/app/i18n/locales/en.json

```diff
@@ -27,6 +27,26 @@
     },
     "cta": "Get the deal"
   },
+  "notFound": {
+    "header": "Error 404",
+    "title": "Nothing to see here...",
+    "description": "The page you are looking for could not be found.",
+    "card1": {

```

Comment:
French and Spanish translations are missing

Reply:
For the moment it will only be in English because the website has the same behavior.

Reply:
So, as English is the fallback lang, it always appears in english for the time being.